### PR TITLE
Add FeedbackSize to SymmetricAlgorithm

### LINKED ISSUE #12368: Port SymmetricAlgorithm FeedbackSize, FeedbackSizeValue, ValidKeySize members
```
F:FeedbackSizeValue
P:FeedbackSize
M:ValidKeySize(System.Int32)
```
In addition ns2.0 related work on AesManaged (to wrap\proxy the method)
```
AesManaged.FeedbackSize
```

and non-ns2.0 work to wrap\proxy FeedbackSize on other types including:
```
RijndaelManaged
RijndaelImplementation
AesCryptoServiceProvider
TripleDESCryptoServiceProvider
```

Unless we want to throw PNSE, FeedbackSize should flow all the way to the provider, and tests should be written for those providers that support something other than 8bytes\64bits.

### PR DESCRIPTION
**Pull request by alexperovich**
Fixes #12368

### CODE REVIEW

File: src/System.Security.Cryptography.Primitives/tests/SymmetricAlgorithm/Trivial.cs [hunk 1]
```diff
@@ -9,6 +9,19 @@ namespace System.Security.Cryptography.Encryption.Tests.Symmetric
...
+        [Theory]
+        [InlineData(-1)]
+        [InlineData(0)]
+        [InlineData(7)]
+        [InlineData(48)]
+        public static void TestBadFeedbackSize(int size)
+        {
+            using (var t = new Trivial { BlockSize = 40 })
+            {
+                Assert.Throws<CryptographicException>(() => t.FeedbackSize = size);
```
Comment (bartonjs) [inline src/System.Security.Cryptography.Primitives/tests/SymmetricAlgorithm/Trivial.cs]: How about a test that shows that if `FeedbackSize`  was set as a large/legal value, then block size was reduced, that there's no (immediate) exception or alteration of the FeedbackSize value?

Or that if FeedbackSizeValue was itself set to a non-mod-8 value that the getter happily returns it without throwing.

**Comment by alexperovich** [comment]
@dotnet-bot test this please

File: src/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs [hunk 2]
```diff
@@ -11,13 +11,14 @@ namespace System.Security.Cryptography
     public abstract partial class Aes : System.Security.Cryptography.SymmetricAlgorithm
     {
         protected Aes() { }
-        public static System.Security.Cryptography.Aes Create() { throw null; }
-        public static System.Security.Cryptography.Aes Create(string algorithmName) { throw null; }
+        public static new System.Security.Cryptography.Aes Create() { throw null; }
```
Comment (alexperovich) [inline src/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs]: the build fails now because it isn't there.

Comment (steveharter) [inline src/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs]: Note I also ran into this build issue locally (before seeing this PR) and just fixed up the Algorithms contracts and merged it (to add the new modifier to all such static Create methods)
https://github.com/dotnet/corefx/commit/ad0c8cd828c5975991107487a5fed8f578892a2a

Comment (alexperovich) [inline src/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs]: ill revert this then

File: src/System.Security.Cryptography.Primitives/tests/SymmetricAlgorithm/Trivial.cs (hunk 1 above)
Comment (alexperovich) [inline src/System.Security.Cryptography.Primitives/tests/SymmetricAlgorithm/Trivial.cs]: Ill add them

File: src/System.Security.Cryptography.Primitives/tests/SymmetricAlgorithm/Trivial.cs [hunk 3]
```diff
@@ -3,12 +3,46 @@
...
+                t.BlockSize = 40;
+            }
+        }
+
+        [Fact]
+        public static void TestFeedbackSizeDoesntValidateFieldValue()
+        {
+            using (var t = new Trivial())
+            {
+                typeof(SymmetricAlgorithm).GetField("FeedbackSizeValue", BindingFlags.NonPublic | BindingFlags.Instance).SetValue(t, 9);
```
Comment (bartonjs) [inline src/System.Security.Cryptography.Primitives/tests/SymmetricAlgorithm/Trivial.cs]: Why not just add SetFeedbackSize method to Trivial?

File: src/System.Security.Cryptography.Primitives/tests/SymmetricAlgorithm/Trivial.cs [hunk 4]
```diff
@@ -3,12 +3,46 @@
...
+        }
+
+        [Fact]
+        public static void TestFeedbackSizeDoesntErrorWhenBlockSizeChanged()
+        {
+            using (var t = new Trivial { BlockSize = 104 })
+            {
+                t.FeedbackSize = 96;
+                t.BlockSize = 40;
+            }
```
Comment (bartonjs) [inline src/System.Security.Cryptography.Primitives/tests/SymmetricAlgorithm/Trivial.cs]: Assert.Equals(96, t.FeedbackSize);?

Or is it 40 now?  (I assume 96)

Comment (alexperovich) [inline src/System.Security.Cryptography.Primitives/tests/SymmetricAlgorithm/Trivial.cs]: It would be 96.

**Comment by alexperovich** [comment]
@dotnet-bot test Innerloop Windows_NT Debug Build and Test please

**Comment by alexperovich** [comment]
@dotnet-bot test Innerloop Windows_NT Release Build and Test please

**Comment by alexperovich** [comment]
@dotnet-bot test Innerloop Windows_NT Debug Build and Test please

**Comment by alexperovich** [comment]
@dotnet-bot test Innerloop Windows_NT Release Build and Test please

**Pull request merged**